CHANGELOG.md

```diff
@@ -41,6 +41,11 @@ a breaking change for the clients.
 - Split `Response` into `InitResponse` and `HandleResponse`; split
   `ContractResult` into `InitResult` and `HandleResult`.
 - Create explicit `QueryResponse`, analogue to `InitResponse` and `HandleResponse`.
+- Add new `ApiResult` and `ApiError` types to represent serializable counterparts to
+`errors::Result` and `errors::Error`. This has a powerful `Into` and `From` implementations
+to transfer back and forth between `Result` and `ApiResult`.
```

Comment:
The conversion `Result` -> `ApiResult` is loosing significan information. How can this be a "back and forth" implementation?

Reply:
I'll clarify.

Reply:
Reworded in https://github.com/CosmWasm/cosmwasm/pull/197/commits/3da90297911cc99c97cf407d96adb9c8bc18aae8